exonum-java-binding/CHANGELOG.md

```diff
@@ -98,6 +98,13 @@ stable Exonum release. See [release notes][exonum-1.0.0-rc.1] for details.
   Any exceptions thrown in these methods are saved in the blockchain
   in `Blockchain#getCallErrors` and can be retrieved by any services or
   light clients.
+- Replaced `BlockchainData` argument in transaction-like `Service` and 
+  `Configurable` methods with `TransactionContext`. The `BlockchainData`
+  remains accessible via `TransactionContext#getBlockchainData`,
+  and service data via `TransactionContext#getServiceData`.
+  <!-- TODO: Shall we rename it into `ExecutionContext` (as
+  TransactionExecutionException has become ExecutionException)?
+  -->
```

Comment:
# ⚠️

Reply:
I like renaming to `ExecutionContext`. For example, `TransactionContext` at `initialize` leads to a question "what kind of transaction do I have here?".

Reply:
OK, thanks, will do here.